SR2020/2020-03-05-coronavirus.md

```diff
@@ -0,0 +1,21 @@
+---
+to: sr2020-teams
+subject: Update on Coronavirus and SR
+---
+
+# Update on SR and the COVID-19 Coronavirus
+
+Hello everyone, 
+
+TL;DR: We're not intending to cancel the event, but will be monitoring the situation closely and will be taking action both during and before the event to minimize risk.
+
+I'm sure you've heard about the threat caused by the COVID-19 coronavirus. and you've likely heard of many large events being cancelled due to concerns of spreading of the virus.
+
+At the current time we're *not* intending to cancel Student Robotics 2020 competition. However, we're keeping a keen eye on the situation, and will be following the advice of the [UK Department of Health and Social Care][uk-coronavirus-page]. We are also working on a back-up plan should the event be unable to run. Considering this is an event for minors, we will be taking decisions with an abundance of caution.
```

Comment:
Change intro and bold

```suggestion
At this time, we're **not** intending to cancel the Student Robotics 2020 competition. However, we're keeping a keen eye on the situation, and will be following the advice of the [UK Department of Health and Social Care][uk-coronavirus-page]. We are also working on a back-up plan should the event be unable to run. Considering this is an event for minors, we will be taking decisions with an abundance of caution.
```

Reply:
> We are also working on a back-up plan should the event be unable to run.

This is news to me? And something which I doubt is necessary.

Reply:
I would suggest removing the backup plan line